synology/start_tunnel_to_server.sh

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env bash
+
+zabbix_server_port="${1}"
+priv_key="${2}"
+
+if [ -z "${zabbix_server_port}" ]  || [ -z "${priv_key}" ]; then
+  echo "Usage:"
+  echo "  ${0} <zabbix_server_port> <priv_key>"
+  exit 1
+fi
+
+if [ -z "${tmp_dir}" ]; then
+  tmp_dir=$(mktemp -d)
+fi
```

Comment:
There was a part underneath here to deal with the public keys of the relays, you will need this.